walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -28,6 +28,16 @@ public void testGreeting() {
 
     String greeting = greeter.greet("Ada");
 
+    Assert.assertEquals("Hello Ada", greeting);
+  }
+
+	@Test
```

Comment:
nit: Let's fix the alignment for this `@Test` annotation.

Reply:
fixed